pages/Magical-Gadgets.md

```diff
@@ -0,0 +1,32 @@
+The Magical Gadgets contains various gadgets with magical powers. Use your magical items to create these!
+
+## Table of contents
+----
+[Ender Backpack](https://github.com/Slimefun/Slimefun4/wiki/Ender-Backpack)  
+[Magic Eye of Ender](https://github.com/Slimefun/Slimefun4/wiki/Magic-Eye-of-Ender)  
+[Elemental Staff](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Wind](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Water](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Fire](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Storm](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
```

Comment:
The plural of "Staff" is "Staves", not "Staffs".

Reply:
Ive been living a lie